covid-vue-app/src/components/Home.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <div class="home">
```

Comment:
Why is this named `Home.vue`? It's it just chart?

Reply:
I don't need it so I'll delete it